app/infrastructure/services/webhook_dispatcher.py

```
@@ -0,0 +1,149 @@
+        """POST to target_url with signature; retry with backoff. Returns True if 2xx."""
+        signature = _sign(sub.secret, body)
+        headers = {
+            "Content-Type": "application/json",
+            "X-Timeline-Signature": f"sha256={signature}",
+        }
+        last_error: Exception | None = None
+        for i, delay in enumerate(RETRY_DELAYS):
+            try:
+                r = await client.post(sub.target_url, content=body, headers=headers)
+                r.raise_for_status()
+                return True
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)